test/node/conversation-group.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let user;
+let client2;
+let user2;
+let createdConversation;
+describe('Group Conversation', () => {
+
+    before(async () => {
+        client = new Circuit.Client(config.bot1);
+        user = await client.logon();
+        client2 = new Circuit.Client(config.bot2);
+        user2 = await client2.logon();
+    });
+
+    after(async () => {
+        await client.logout();
+        await client2.logout();
+    });
+
+    it('should create a group conversation', async () => {
+        const topic = `${Date.now()}a`;
+        const res = await Promise.all([
+            client.createGroupConversation([user2.userId], topic),
+            helper.expectEvents(client, [{
+                type: 'conversationCreated',
+                predicate: evt => evt.conversation.topic === topic && evt.conversation.participants.includes(user.userId) && evt.conversation.participants.includes(user2.userId)
+            }])
+        ]);
+        createdConversation = res[0];
+        assert(createdConversation && createdConversation.participants.includes(user.userId) && createdConversation.participants.includes(user2.userId));
+    });
+
+    it('should get the group conversation by its Id', async () => {
```

Comment:
We don't really need this since we already test the getConversationById in the direct conv test.